widget.json

```diff
@@ -19,7 +19,8 @@
     ],
     "assets": [
       "dist/css/index.css",
-      "dist/app.js"
+      "dist/app.js",
+      "vendor/hichartsExportModule.js"
```

Comment:
shouldn't vendor libs being added before the app? cc @tonytlwu

Reply:
Oh yes, definitely